sample/conf/mainnet-mockminer-conf.toml

```diff
@@ -8,4 +8,5 @@ prometheus_bind = "0.0.0.0:9153"
 
 [burnchain]
 mode = "mainnet"
-peer_host = "127.0.0.1"
+peer_host = "192.168.86.251" #start9
```

Comment:
Artifact?

Reply:
indeed, thanks for catching! i was digging into `./docs/profiling.md` and needed a local btc node on my network to test with. removed